### ISSUE TITLE
Try-builds fail to run if installed on touch capable systems

### ISSUE THREAD

**Issue by XLTechie** [nvaccess/nvda#12563]
### Steps to reproduce:

On a touchscreen equiped system, download a try-build from a recent PR and install it. Try to start NVDA.

### Actual behavior:

A try-build from a recent PR (#12560), which runs successfully while installed on a non-touch system, fails after installation on a touch-capable system.
It runs fine as a temporary copy.

```
Initializing touchHandler
DEBUG - touchHandler.initialize (05:06:52.773) - MainThread (26748):
Touchscreen detected, maximum touch inputs: 256
CRITICAL - __main__ (05:06:52.783) - MainThread (26748):
core failure
Traceback (most recent call last):
  File "nvda.pyw", line 359, in <module>
  File "core.pyc", line 628, in main
  File "touchHandler.pyc", line 342, in initialize
  File "touchHandler.pyc", line 322, in setTouchSupport
  File "touchHandler.pyc", line 217, in __init__
  File "touchHandler.pyc", line 232, in run
  File "_ctypes/callproc.c", line 935, in GetResult
PermissionError: [WinError -2147024891] Access is denied
```

This was from a try-build based off of current beta branch as of 6/18/2021.

[Full log](https://github.com/nvaccess/nvda/files/6676623/nvda.log)

### Expected behavior:

NVDA continues to run after installation on touch capable systems.

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

A try-build based on beta.

#### Windows version:

1909

#### Name and version of other software in use when reproducing the issue:

N/A

#### Other information about your system:

The user account on which NVDA is (trying to) run is not an admin account.

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

N/A

#### If add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

N/A

**Comment by lukaszgo1** [comment]
I believe this should be fixed by #12447 which is currently in master.

**Comment by XLTechie** [comment]
Ah, my bad. I remembered that one but thought it had made it into beta.
